Backend/src/model/date.py

```diff
@@ -1,20 +1,20 @@
 from __future__ import annotations
-from .dayOfWeek import DayOfWeek
+from .day_of_week import DayOfWeek
 
 class Date:
 
-    def __init__(self, day: DayOfWeek, startTime: str, endTime: str) -> None:
+    def __init__(self, day: DayOfWeek, start_time: str, end_time: str) -> None:
```

Comment:
The start time and end time strings are currently stored like this: "14:00 - 16:30", would you prefer if we split them into two separate strings? Resulting in something like this: {"DayOfWeek": "Tue", "StartTime": "14:00", "EndTime": "16:30" }?